app/controllers/comments_controller.rb

```diff
@@ -10,6 +10,7 @@ def create
     @comment.user = current_user
     if @comment.save
       track_created(@comment)
+      broadcast_comment
```

Comment:
I have doubts about this:
We create notifications on a bg job when calling `track_created`.
Then in the next line we call `broadcast_comment` which calls `@comment.notifications`.
Those `notifications` may still not exist, depending on the bg job.

Reply:
I think the first one, rendering from outside the controller, at least should work.

I also think that if we have moved creating activities and notifications to a bg job, it also makes sense to send those notifications from the bg job.